compute/encryption/generate_wrapped_rsa_key_test.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import generate_wrapped_rsa_key
+
+
+def test_main():
+    generate_wrapped_rsa_key.main()
```

Comment:
Good for initial test. How about to write an e2e test that performs one of the operations listed on:
https://cloud.google.com/compute/docs/disks/customer-supplied-encryption
and runs it on jenkins?